GVFS/GVFS.Common/Database/Placeholders.cs

```diff
@@ -0,0 +1,199 @@
+﻿using Microsoft.Data.Sqlite;
+using System;
+using System.Collections.Generic;
+using System.Data;
+
+namespace GVFS.Common.Database
+{
+    public class Placeholders : IPlaceholderDatabase
+    {
+        private GVFSDatabase database;
+
+        public Placeholders(GVFSDatabase database)
+        {
+            this.database = database;
+        }
+
+        public static void CreateTable(SqliteCommand command)
+        {
+            command.CommandText = @"CREATE TABLE IF NOT EXISTS [Placeholders] (path TEXT PRIMARY KEY, pathType TINYINT NOT NULL, sha char(40) ) WITHOUT ROWID;";

```

Comment:
Style comment:  It looked a bit weird to me to have `CreateTable` accept a `SqliteCommand`, I would have expect `CreateTable` to create the `SqliteCommand` that it needs.

What would you think of having the method take a `SqliteConnection` instead?